tests/scripts/depends.py

```diff
@@ -261,7 +261,10 @@ def test(self, options):
                          'MBEDTLS_ENTROPY_FORCE_SHA256',
                          'MBEDTLS_SHA256_USE_A64_CRYPTO_IF_PRESENT',
                          'MBEDTLS_SHA256_USE_A64_CRYPTO_ONLY'],
-    'MBEDTLS_X509_RSASSA_PSS_SUPPORT': []
+    'MBEDTLS_X509_RSASSA_PSS_SUPPORT': [],
+    # The support for TLS 1.3 is not complete yet and in particular
+    # pre-shared keys are not supported.
```

Comment:
PSK are actually supported, the comment about that in `mbedtls_config.h` is outdated.